starter-project-web-vue2/components/ruth/RuthBlog.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <div>
+    <nuxt-link
```

Comment:
It'll be better if the click event is attached with the title or edit button. since there's also a delete and edit button. those events may propagate to parent

Reply:
Alright, done.